readthedocs/core/views/serve.py

```diff
@@ -175,7 +175,7 @@ def _serve_file(request, filename, basepath):
     :raises: ``Http404`` on ``UnicodeEncodeError``
     """
     # Serve the file from the proper location
-    if settings.DEBUG or settings.PYTHON_MEDIA:
+    if (settings.DEBUG or settings.PYTHON_MEDIA) and not settings.RTD_DOCKER_COMPOSE:
```

Comment:
Is this to enable DEBUG to be true?

Reply:
Right. Depending on what you are working on, you will want `DEBUG=True` (e.g. something with the dashboard) or you will want `DEBUG=False` (e.g. working on custom 404s) but still, in both cases you want to serve files from NGINX.